openqabot/openqabot.py

```diff
@@ -61,19 +62,18 @@ def __call__(self):
             post += worker(self.incidents, self.token, self.ci, self.ignore_onetime)
 
         if self.dry:
-            log.info("Would trigger %d products in openQA" % len(post))
+            log.info("Would trigger %d products in openQA", len(post))
             for job in post:
                 log.info(job)
 
         else:
-            log.info("Triggering %d products in openQA" % len(post))
+            log.info("Triggering %d products in openQA", len(post))
             for job in post:
-                log.info("Triggering %s" % str(job))
+                log.info("Triggering %s", str(job))
                 try:
                     self.post_openqa(job["openqa"])
                 except PostOpenQAError:
                     log.info("POST failed, not updating dashboard")
-                    pass
```

Comment:
please check, seems to me not useful

Reply:
change is good. `pass` should be used if there is no other execution in branch but with the log call there is, so no need for the pass.